Onepunch/README.md

```diff
@@ -1,26 +1,37 @@
 # Onepunch
+
 ## Screenshots
-![Home](./__Home.png)
-![Profile](./__Profile.png)
-![Playlist](./__Playlist.png)
+
+![Home_legacy](./__Home.png)
+
+<table>
+  <tr>
+    <td><img src="./__Playlist.png" alt="2" width = 360px> Playlist_legacy </td>
+    <td><img src="./__Profile.png" alt="3" width = 360px> Profile_legacy </td>
```

Comment:
Remove the text, otherwise it is shown next to the image

Reply:
ohh my bad.. it was showing okay in my repo. I'll do it.

Reply:
changed from html to default GitHub markdown table. hope this works